Changelog.md

```diff
@@ -11,10 +11,11 @@
 
 ### Tests
 
-- runs nightly test of main branch of odo against [SBO master branch](https://github.com/redhat-developer/service-binding-operator) ([4942](https://github.com/openshift/odo/issues/4942))
+- runs nightly test of main branch of odo against [SBO master branch](https://github.com/redhat-developer/service-binding-operator) ([#4942](https://github.com/openshift/odo/issues/4942))
```

Comment:
?

Reply:
What is this change? It is completely unrelated to this PR

Reply:
https://github.com/openshift/odo/pull/4982/commits/1f0ac44c04eb7774f29f013d3e3c393dc9c33887

I hope that helps.

https://github.com/openshift/odo/pull/4983

I hope that helps as well.